vars/withGithubStatus.groovy

```diff
@@ -31,23 +31,41 @@ def call(Map args = [:], Closure body) {
   def description = args.get('description', context)
   def tab = args.get('tab', 'pipeline')
   def isBlueOcean = args.get('isBlueOcean', false)
+  def ignoreGitHubFailures = args.get('ignoreGitHubFailures', true)
 
   def redirect = detailsURL(tab: tab, isBlueOcean: isBlueOcean)
 
   try {
-    notify(context, "${description} ...", 'PENDING', redirect)
+    notifyMap(context: context, description: "${description} ...", status: 'PENDING', redirect: redirect, ignoreGitHubFailures: ignoreGitHubFailures)
     withAPM(){
       body()
     }
-    notify(context, "${description} passed", 'SUCCESS', redirect)
+    notifyMap(context: context, description: "${description} passed", status: 'SUCCESS', redirect: redirect, ignoreGitHubFailures: ignoreGitHubFailures)
   } catch (err) {
-    notify(context, "${description} failed", 'FAILURE', redirect)
+    notifyMap(context: context, description: "${description} failed", status: 'FAILURE', redirect: redirect, ignoreGitHubFailures: ignoreGitHubFailures)
     throw err
   }
 }
 
+// This is consumed by elastic/kibana.git@master:.ci/end2end.groovy
 def notify(String context, String description, String status, String redirect) {
-  retryWithSleep(retries: 2, seconds: 5, backoff: true) {
-    githubNotify(context: "${context}", description: "${description}", status: "${status}", targetUrl: "${redirect}")
+  notifyMap(context: context, description: description, status: status, redirect: redirect)
```

Comment:
Here the `ignoreGitHubFailures` is not passed so it's expected to pass the default value 'true', but in L64, the default value if not present is false. Is this on purpose?

Reply:
```suggestion
  // Fail in case the GitHub API calls fail for any reason
  notifyMap(context: context, description: description, status: status, redirect: redirect, ignoreGitHubFailures: false)
```